var/spack/repos/builtin/packages/riscv-gnu-toolchain/package.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import shlex
+from subprocess import Popen
+
+from spack.package import *
+
+
+class RiscvGnuToolchain(AutotoolsPackage):
+    """A cross-compilation tool for RISC-V."""
+
+    homepage = "https://spack-tutorial.readthedocs.io/"
+    git = "https://github.com/riscv-collab/riscv-gnu-toolchain.git"
+
+    phases = ["configure", "build"]
+
+    maintainers = ["wanlinwang"]
+
+    version("develop", branch="master", submodules=True)
+    version("2022.08.08", tag="2022.08.08", submodules=True)
+
+    # Dependencies:
+    depends_on("pkg-config", type=("build"))
+    depends_on("autoconf", type=("build"))
+    depends_on("automake", type=("build"))
```

Comment:
If `configure` is distributed with the package, then these aren't necessary. If it's not, then you need an `autoreconf` phase added to the package (there's default logic for that).

For many packages, they use `depends_on('autoconf', when='@main:', type='build')` because they only need to autoreconf for dev versions.